kalite/updates/management/commands/videodownload.py

```diff
@@ -175,6 +176,14 @@ def youtube_dl_cb(stats, progress_callback, *args, **kwargs):
                             progress_callback(percent=percent)
                         scrape_video(video.youtube_id, quiet=not settings.DEBUG, callback=partial(youtube_dl_cb, progress_callback=progress_callback))
 
+                    except IOError as e:
+                        logging.exception(e)
+                        video.download_in_progress = False
+                        video.save()
+                        failed_youtube_ids.append(video.youtube_id)
+                        time.sleep(10)
```

Comment:
@rtibbles 
I don't know whether we should use sleep or
 we can use am_i_online function and keep waiting until internet is available again.